.github/workflows/coverage.yml

```diff
@@ -0,0 +1,72 @@
+name: Code Coverage Update
+
+# read-write repo token
+# access to secrets
```

Comment:
This runs in its own context that has access to the repo. It cannot be changed by a PR directly so it's not a problem. A PR changing this would have to be merged first (like this one).